Formula/deployer.rb

```diff
@@ -0,0 +1,16 @@
+class Deployer < Formula
+  desc "Deployment tool written in PHP with support for popular frameworks"
+  homepage "https://deployer.org"
+  url "https://deployer.org/releases/v6.1.0/deployer.phar"
+  sha256 "260a88bb89711cb2251496e8c353ee3be0e86cf8985117a8a714369fb35699f8"
+
+  bottle :unneeded
+
+  def install
+    bin.install "deployer.phar" => "deployer"
+  end
+
+  test do
+    system "#{bin}/deployer", "help"
```

Comment:
Please provide a more functional test. Could we run `dep init` and `dep list`, and check that the expected file(s) is(are) created?

Reply:
Done